Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -42,9 +213,78 @@ def test_module(client: PolicyComputeEngine) -> str:
     raise ValueError("Failed to establish connection with provided credentials.")
 
 
+def illumio_traffic_analysis_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> CommandResults:
+    """Retrieve the traffic for a particular port and protocol.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: arguments obtained from demisto.args()
+
+    Returns: CommandResult object
+    """
+    port = arg_to_number(args.get('port'))
+    protocol = args.get('protocol', '').lower()
+    start_time = arg_to_datetime(args.get('start_time')).isoformat()  # type: ignore
+    end_time = arg_to_datetime(args.get('end_time')).isoformat()  # type: ignore
+    policy_decisions = argToList(args.get('policy_decisions'))
+    validate_required_parameters(port=port, protocol=protocol, start_time=start_time, end_time=end_time,
+                                 policy_decisions=policy_decisions)
+    validate_arguments_for_traffic_analysis(port, policy_decisions, protocol)  # type: ignore
+    query_name = 'Traffic analysis'
```

Comment:
Come up with something more descriptive here - may also want to make it unique.

Reply:
Alright, I'll change it